platform/pytorch-cpu/build.sh

```diff
@@ -3,5 +3,16 @@ export NO_CUDA=1
 export CMAKE_PREFIX_PATH=$PREFIX
 
 export VERBOSE=1
-patch -p1 -d third_party/gloo < ${RECIPE_DIR}/gloo-SPEED_UNKNOWN.patch
 $PYTHON setup.py install
+
+{
+  {
+  mkdir build_libtorch
+  cd build_libtorch
+  BUILD_TORCH=ON ONNX_NAMESPACE=onnx_torch bash ../tools/build_pytorch_libs.sh --use-nnpack caffe2
+  } && {
+  echo -e "\nSuccessfully built libtorch at `dirname $PWD`/torch/lib/tmp_install\n"
```

Comment:
Hmm. but why not install libtorch to the proper directory? Is it because there is a conflict?

Reply:
It seems that in fact it's also built in `torch/`, and they've removed `/torch/lib/tmp_install` in a later commit. I just tried the latest version, and pytorch itself was built successfully. But they changed the way to build libtorch, and there's a bug that I don't know how to fix. I'm gonna ask them before updating this PR.